service/src/main/java/org/cbioportal/service/util/AlterationEnrichmentUtil.java

```diff
@@ -189,7 +192,7 @@ public long includeFrequencyForSamples(
 
     public long includeFrequencyForPatients(
             List<MolecularProfileCaseIdentifier> molecularProfileCaseIdentifiers,
-            List<T> alterationCountByGenes,
+            List<T> AlterationCountBases,
```

Comment:
same here, if we can't change it, it's fine too

Reply:
See my previous comment. But I agree that this name at least should be `alterationCounts` as also used in -includeFrequencyForSamples_.